README.md

```diff
@@ -309,13 +310,27 @@ A certificate may be supplied in the manner described above and can be used to p
 
 Here is an example of a Kubernetes secret bearing a certificate for use with a specific fully-qualified domain name.  The following criteria must be met:
 
-* Must be named `<domain>-cert`
+* Secret name must be for the form `<arbitrary name>-cert`
+  * This must be associated to the domain using the [router.deis.io/certificates](#certificates-annotation) annotation.
```

Comment:
Ahh I see. In ReST you had to use a new symbol for more indentation levels, but I guess that's not the case here.